src/test/scala/ing/wbaa/druid/DruidQuerySpec.scala

```diff
@@ -233,13 +234,8 @@ class DruidQuerySpec extends WordSpec with Matchers with ScalaFutures {
           LongSumAggregation(name = "count", fieldName = "count")
         ),
         postAggregations = List(
-          ArithmeticPostAggregation(name = "halfCount",
-                                    fn = "/",
-                                    fields = (
-                                      FieldAccessPostAggregator(name = "count",
-                                                                fieldName = "count"),
-                                      ConstantPostAggregator(name = "two", value = 2)
-                                    ))
+          (FieldAccessPostAggregation("count") / ConstantPostAggregation("two", 2))
```

Comment:
This looks much much better now. Kudos!

Reply:
Basically, we can do even this 
```
"count" / 2
```
making the string as `FieldAccessor` and int as `Constant` (omitting name of constant because it is of no use there, right).
But I d rather leave it as it is, because too much "magic" could be confusing.